pkg/ottl/grammar.go

```diff
@@ -270,6 +271,19 @@ type list struct {
 	Values []value `parser:"'[' (@@)* (',' @@)* ']'"`
 }
 
+type mapValue struct {
+	Values []mapItem `parser:"'{' (@@ ','?)* '}'"`
+}
+
+type mapItem struct {
+	Key   *string `parser:"@String ':'"`
+	Value *value  `parser:"@@"`
+}
+
+func (m *mapItem) GoString() string {
```

Comment:
Was this just for debugging, or can it be used for grammar parsing errors?

Reply:
good catch, that is a leftover from doing some debugging - will remove that again